pkgs/development/python-modules/torch/binary-hashes.nix

```diff
@@ -6,66 +6,86 @@
 # To add a new version, run "prefetch.sh 'new-version'" to paste the generated file as follows.
 
 version : builtins.getAttr version {
-  "1.13.1" = {
+  "2.0.0" = {
     x86_64-linux-38 = {
-      name = "torch-1.13.1-cp38-cp38-linux_x86_64.whl";
-      url = "https://download.pytorch.org/whl/cu117/torch-1.13.1%2Bcu117-cp38-cp38-linux_x86_64.whl";
-      hash = "sha256-u/lUbw0Ni1EmPKR5Y3tCaogzX8oANPQs7GPU0y3uBa8=";
+      name = "torch-2.0.0-cp38-cp38-linux_x86_64.whl";
+      url = "https://download.pytorch.org/whl/cu118/torch-2.0.0%2Bcu118-cp38-cp38-linux_x86_64.whl";
```

Comment:
Had another question. It seems to me that the binaries here would only work when `torch` is used with `CUDA` 11.8, is that right?

I tried to build it with cuda 11.7 and it can build and run

```
>>> import torch
>>> torch.version.cuda
'11.7'
```

Can this be a potential problem, if the original torch binary is built from CUDA 11.8?

Thanks!

```

Reply:
Thanks for the explanation, Junji!